Sidney-/slugtrade
# removed weird HTML-tag-like files from repo

Removed eight empty files from "slugtrade/slug_trade":  "n-top.pgwSlider", "ul", "uldth.pgwSlider", "li", "r", "usplay.pgwSlider","li,", and "li:last-child,".

How to test:
- make sure app still starts

Note: Without knowing what the intended purpose of these files was or why they were added the repo, the only way to fully test this fix is to ensure the entire app behaves identically as before the fix, which is unfeasible without automated tests. Probably, though, the files have no effect on the project, so I figure that if the app doesn't crash that's sufficient testing.
